data/human/news.txt

```diff
@@ -2150,3 +2150,313 @@ news "graffiti on spaceport wall"
+		"with substantial sections of hull heavily battered or missing altogether. Fair guess they didn't have enough anti-missle. "
+	word
+		"As it comes to a stop in the maitenance bay engineering shuttles begin "
+	word
+		"buzzing "
+		"fluttering "
+	word
+		"around it, assessing the damage and beginning the work on repairs. "
+	word
+		"The work won't take the shuttles long."
+		"They'll be working for the rest of the shift on this one."
+
+news "Symbol War"
+	location
+		planet Gagarin
+name
+	"Guerrilla Symbology"
+message
+	word
+		"It's an average day in a busy spaceport, "
+	word
+		"there's no real quiet anywhere in the port, "
+		"it's the expected amout of noisy crowd background noise, "
+		"with the normal buzz of activity with thousands of things happening every second, "
+	word
+		"it can be a bit much but anyone who works with ships learns to tune it out. "
+	word
+		"In spite of the drive to tune out the rumble of the crowds "
+		"However, against the drive to tune out the crowds "
+		"A rare and fleeting drop in the rumble allows% "
+	word
+		"you hear something that stands out "
+	word
+		"turning your attention to the unexpected noise, "
+		"glancing over to the new commotion, "
+	word
+		"you see a squad of "
+	word
+		"CCOR irregulars "
+		"CCOR guerrillas "
+	word
+		"a few of them are armed but most have paint, plaster, and ladders."
+	word
+		"They "
+	word
+		"leave the spaceport traffic alone,"
+		"aren't disrupting traffic,"
+	word
+		"rather "
+		"instead "
+	word
+		"they are "
+	word
+		"climbing the back wall toward the giant crest of the planetary government% "
+		"walking along the peripheral walls of the port focusing on the small government crests "
+		"returning to CCOR crests that have been covered with planetary government crests "
+	word
+		"they dilligently replace the government crests with CCOR's. "
+	word
+		"Spaceport security are "
+	word
+		"noticably absent."
+		"actively ignoring them."
+		"keeping their distance."
+	replace
+		"% you" " you to"
+		"% they dilligently replace the government crests" " they dilligently replace the large crest"
+		"%" ""
+
+news "IP Theft"
+	location
+		shipyard "Betelgause Basics" "Betelgause Advanced"
+		not government "Pirate"
+name
+	"Bootlegged Ships"
+message
+	word
+		"A poster on the spaceport wall reads "
+		"An audio broadcast playing in the spaceport recites "
+		"A televised message on one of the news screens broadcasts "
+	word
+		"'A public-service announcement from Betelgeuse Shipyards: How do you know that a Betelgause Ship TM"
+	word
+		"has the time-tested quality? "
+		"will live up to your high expectations? "
+		"is the genuine article? "
+	word
+		"It's quite simple! Ensure that when buying new you do so from a reputable dealer or if you're interested in one that has itself withstood the test of time request the certificate of authenticity from the seller with your purchace. These simple precautions will ensure that "
```

Comment:
```suggestion
		"It's quite simple! Ensure that when buying new you do so from a reputable dealer or if you're interested in one that has itself withstood the test of time request the certificate of authenticity from the seller with your purchase. These simple precautions will ensure that "
```
This could use some rephrasing, too.